packages/mui-joy/test/typescript/moduleAugmentation/CssVarsProvider.spec.tsx

```diff
@@ -106,11 +106,11 @@ declare module '@mui/joy/styles' {
       xxxs: '',
     },
     typography: {
-      ads: (vars) => ({
-        fontFamily: vars.fontFamily.secondary,
-        fontSie: '',
+      ads: {
```

Comment:
Why was this changed?

Reply:
Because now the typography system accepts a plain object, not a function.